docs/rocketchat/get-help-in-rocketchat.md

```diff
@@ -1,95 +1,100 @@
 
-keywords: Rocket.chat, help in rocket.chat, support, help, community, developers, help, cluster issue, tool issue, stack issue, problem solving, 
 
-page_purpose: Helps developer find the best way to troubleshoot issues with clusters,  tools, or stacks
+> **route.yml YAML**
 
-audience: technical lead, openshift 101 students, openshift 201 students,  developers
+    kind: Route
+    apiVersion: route.openshift.io/v1
+    metadata:
+     name: my-web-server
+     namespace: abcd12-dev
+     labels:
+      app: my-web-server
+      group: my-project
+    spec:
+     host: my-web-server-abcd12-dev.apps.silver.devops.gov.bc.ca
+     to:
+      kind: service
+      name: my-web-server
+      weight: 100
+     port:
+      targetPort: web
+     tls:
+      termination: edge
+      insecureEdgeTerminalPolicy: Redirect
+     wildcardPolicy: None
 
-author: Alex Lloyd
+The user tells us the where, when, what, and how of an issue. We want to know what you expected, what happened, and in what environment, the problem happened. Before connecting with the community, check [Jon Skeet's checklist](https://codeblog.jonskeet.uk/2012/11/24/stack-overflow-question-checklist/) if you've done everything possible before posting to Rocket.Chat.
 
-editor: Pilar Solares
+Below, we offer more guidance to strengthen a question. Strong questions simplify the response time from a platform community member. 
 
-content_owner: Olena Mitvoska
+## The 'how' and 'what'
 
-sort_order: 4
----
+### How
 
-# Get help in Rocket.Chat
-Last updated: **July 10, 2023**
+Please tell us the problem, what you expected when doing a task, and the importance of it.
 
-To get support on Rocket.Chat, you need to ask questions. Including the right information in your questions can help you get a better answer more quickly. 
-Below, we outline the information you should include when asking for help or support from the platform community on Rocket.Chat.
+If it’s a sensitive task, please specify confidentiality is necessary. Being forthcoming about confidentiality stops us from expecting more information.  
 
-## On this page
-* [**Where has the issue ocurred?**](#where-has-the-issue-ocurred)
-* [**What are you trying to do?**](#what-are-you-trying-to-do)
-* [**What is the problem?**](#what-is-the-problem)
-* [**What have you done to fix the problem?**](#what-have-you-done-to-fix-the-problem)
-* [**What kind of help do you want?**](#what-kind-of-help-do-you-want)
-* [**Posting a question**](#posting-a-question)
-* [**Related pages**](#related-pages)
-<!-- ### End of "On this page" -->
+**Please don’t include** sensitive artifacts, like passwords, tokens, API keys, etc.
 
----
-## Where has the issue ocurred?
+### What
 
-**If the problem is with a cluster**
+1. Tell us about the nature of the problem, what it looks like and how it’s behaving. For example, if a screen doesn’t load as you expected, give us a description of what it looks like, and what it should look like.
```

Comment:
```suggestion
1. Tell us about the nature of the problem, what it looks like and how it’s behaving. For example, if a screen doesn’t load as you expected, give us a description of what it looks like, and what it should look like
```

Reply:
what about adding "include a screenshot" as a suggestion here? (maybe it's my front end bias, but this is sometimes so helpful!)